src/config/config.service.ts

```diff
@@ -17,4 +17,9 @@ export class ConfigService {
         password: process.env.MONGO_PASSWORD,
         replicaSet: process.env.MONGO_REPLICASET,
     };
+
+    public jwt = {
+        accessTokenSecret: 'du:<9tv:B{P^X/s~Lv4+MsT7.>%w%!A$zU%_C~Pj3[,]+~c2LB;;fLBE5E@NKFg',
```

Comment:
`process.env.ACCESS_TOKEN_SECRET || 'du:<9tv:B{P^X/s~Lv4+MsT7.>%w%!A$zU%_C~Pj3[,]+~c2LB;;fLBE5E@NKFg'`

Don't forget this is a public repo ^^